dashboard/test/ui/step_definitions/blockly.rb

```diff
@@ -124,6 +124,11 @@
   @browser.execute_script("__TestInterface.loadBlocks('" + arranged_blocks_xml + "');")
 end
 
+And /^I've initialized the workspace with the Calc solution blocks$/ do
+  @browser.execute_script("Blockly.mainBlockSpace.clear();")
+  @browser.execute_script("__TestInterface.loadBlocks(appOptions.level.solutionBlocks);")
```

Comment:
I guess it could work in a lot of places where the solution is defined by the blocks. Maze is one place it wouldn't. THink I should make the text non-Calc specific?